shared/js/media/crop_resize_rotate.js

```diff
@@ -351,12 +347,7 @@ function cropResizeRotate(blob, cropRegion, outputSize, outputType,
     // that we do not want it to decode all of the pixels in the image. The
     // #-moz-samplesize= fragment allows us to specify that JPEG images
     // should be downsampled while being decoded, and this can save a lot of
-    // memory. If we are not going to downsample the image, but are going to
```

Comment:
Comments in Line 199 and Line 212 still mention #xywh media fragment, now that we are removing #xywh media fragment, we can delete those comments.